package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "altis-consent",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

Comment:
Given this isn't an npm package it saves a bit of effort to change the version to `"private": true` - you only have to update the `plugin.php` file version then.